packages/create-project/src/helpers/is-folder-empty.ts

```diff
@@ -0,0 +1,55 @@
+import chalk from 'chalk';
+import fs from 'fs';
+import path from 'path';
+
+export function isFolderEmpty(root: string, name: string): boolean {
```

Comment:
Why don't we just require that the target path doesn't exist?
I'm wondering where this is coming from.

Reply:
I guess it covers the case in which someone habitually starts out a new project by creating a dir? Like:

```
mkdir myapp
```

"okay, now maybe there's a template I can use? oh there's this tool"

```
npx @temporalio/create myapp
```